src/Collapsible/Collapsible.scss

```diff
@@ -1,9 +1,5 @@
-@import "../Card/variables"; // TODO: Remove this dependency
```

Comment:
Are you sure, this import can be removed?

Reply:
Yes, I checked the Collapsible component page after deleted this import